lib/service-test-runner/runner.js

```diff
@@ -16,6 +20,9 @@ class Runner {
    * Prepare the runner by loading up all the ServiceTester objects.
    */
   prepare() {
+    if (this.options.testedServerUrl) {
+      testConfig.tested = this.options.testedServerUrl
```

Comment:
It would be better not to mutate the test config; perhaps better to set it from the environment inside `test-config`, as I mention below.

Reply:
I agree with you, done :-)